pkg/validate/service.go

```diff
@@ -103,17 +105,20 @@ func (s *notaryService) getImageDigestHash(image string) ([]byte, error) {
 	if err != nil {
 		return []byte{}, fmt.Errorf("get image: %w", err)
 	}
-	m, err := i.Manifest()
+
+	hash, err := i.Digest()
+
 	if err != nil {
-		return []byte{}, fmt.Errorf("image manifest: %w", err)
+		return []byte{}, fmt.Errorf("image digest: %w", err)
 	}
 
-	bytes, err := hex.DecodeString(m.Config.Digest.Hex)
+	fmt.Printf("Sha of admitted docker image   : %s \n", hash.Hex)
```

Comment:
I would recommend to introduce a logging API to have an option to enable and disable debug messages

Reply:
[with-zap.tgz](https://github.com/kyma-project/warden/files/10188704/with-zap.tgz)